client/.env.example

```diff
@@ -1 +1,2 @@
-REACT_APP_SERVER_URL=your server url
\ No newline at end of file
+REACT_APP_SERVER_URL=your server url
+REACT_APP_GOOGLE_CLIENT_ID=your google app id
```

Comment:
:+1: 

Didn't know the use of .example

Very useful

Reply:
your all the envs used in production in client side are visible to all users. So you don't need .example env you can directly push your .env file

Reply:
Makes sense, we can have .env.local (for the local environment) and .env (for production).